src/app/app-layout/app-layout.component.html

```diff
@@ -1,16 +1,15 @@
 <div class="flex flex-col h-full bg-blue-50">
-  <div class="flex-grow-0 sticky top-0 h-20">
+  <div class="flex justify-center sticky top-0 bg-white shadow-md">
     <app-navbar></app-navbar>
   </div>
-  <div class="top-4 flex overflow-x-hidden h-full relative ">
+  <div class="top-2 flex overflow-x-hidden h-full relative ">
 
-    <div class="bg-gray-900 absolute w-full h-full z-10
+    <!-- <div class="bg-gray-900 absolute w-full h-full z-10
     transition-all ease-in-out duration-75 opacity-0 invisible"
          [class]="(isOpen$ | async) ? '!visible !opacity-30' : ''"
          (click)="hideNavbar()">
```

Comment:
If this is commented out for a reason, set a short description in TODO comment above:

```html
<!-- TODO: remove this when blablabla -->
<!-- <div>unused code</div> -->
```